RELEASE.md

```diff
@@ -13,6 +13,7 @@ Please follow the established format:
 - Remove metrics plots from metadata panel and add link to the plots on Experiment tracking. (#1268)
 - Link plot and JSON dataset names from experiment tracking to the flowchart. (#1165)
 - Bump minimum version of React from 16.8.6 to 17.0.2. (#1282)
+- Show preview of data in metadata panel. (#1288)
```

Comment:
hey i like the idea of including the "preview of pandas.CSVDataSet and pandas.ExcelDataSet" here too. Even though it will also mention it in our Release highlight in the UI, I guess no harm to mention here again.

I think the React version might be a major change actually but maybe @tynandebold can confirm this?

Reply:
The React version change will make this a major release, so probably ok to leave it where it is.